apps/fulfillment/index.html

```diff
@@ -3,7 +3,6 @@
   <head>
     <meta charset="utf-8" />
     <title>Fulfillment App</title>
-    <base href="/" />
 
     <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
we can keep this at boilerplate or move it to meta. If we decide on it at boilerplate, i'd probably have it in both branches on our boilerplate repo.

Reply:
It's kind of required for minimal PWA setup. In case we move it to `meta` we can have issues with Lighthouse scoring and others.

Reply:
https://github.com/spryker/oryx/pull/916#discussion_r1399244681

Reply:
I'd vote for keeping it as long as we don't have an SSR, so we gonna have different viewport settings until meta service is loaded.

Reply:
I prefer leaving it out, as modern browser will do this default behaviour without this setting anyway. It results in a homogene index.html for all our apps. Going forward, we can introduce app specific meta resolvers for the viewport.